ci-support/conda_upload.sh

```diff
@@ -21,6 +21,7 @@ if [ `uname` == "Linux" ]; then
     conda install -n root gcc future
     which python
     export UVCDAT_ANONYMOUS_LOG=False
+    BRANCH=${TRAVIS_BRANCH}
```

Comment:
are you positive about this? I had issue in the past on travis with this. TRAVIS_BRANCH is set for pull request but not push, or vice versa.